### ISSUE TITLE
[FEATURE] Community Partners (page)

### ISSUE THREAD

**Issue by vinzvinci** [web3community/web3community.github.io#76]
### Detailed description

We need a separate page to showcase our community partners. 

### Context

It is best to do this, so we can show our appreciation to them.

### Additional information

If anyone has a good idea about the design, feel free to comment below.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Panquesito7** [comment]
We can do something similar to what [**this page**](https://heptagram.xyz/heptagram-devs.html) does. 🙂

**Comment by vinzvinci** [comment]
I like it, but I want to know other ideas.

**Comment by shreyaansjain06** [comment]
I want to work on this issue

**Comment by mkubdev** [comment]
Do you have design idea @shreyaansjain06 ?

**Comment by shreyaansjain06** [comment]
Do you have any idea or design on how this page should look

**Comment by vinzvinci** [comment]
We don't have yet @shreyaansjain06. Please help us to find unique design for this

**Comment by shuklaritvik06** [comment]
Yes please share if you have any design idea for that page

**Comment by shreyaansjain06** [comment]
ok

**Comment by shreyaansjain06** [comment]
I think we should do some edit in the design what @Panquesito7 shared

**Comment by vinzvinci** [comment]
I agree with you, you can share figma here (view only). And we can give u feedback

**Comment by shreyaansjain06** [comment]
I have made the figma You can give your feedback
[Link](https://www.figma.com/file/P8ytuqbDBEgsSEJILvsgvV/Untitled?node-id=0%3A1)

**Comment by mkubdev** [comment]
![Capture](https://user-images.githubusercontent.com/50140834/136209558-c252ce7c-1de5-4fc3-9d8d-c780a6bf9cd9.JPG)

Please update permissions. It's ok for you @shuklaritvik06 @vinzvinci  ?

**Comment by shuklaritvik06** [comment]
> ![Capture](https://user-images.githubusercontent.com/50140834/136209558-c252ce7c-1de5-4fc3-9d8d-c780a6bf9cd9.JPG)
> 
> Please update permissions. It's ok for you @shuklaritvik06 @vinzvinci ?

Nope same for me too

**Comment by vinzvinci** [comment]
same here

**Comment by shreyaansjain06** [comment]
sorry i forgot to update permission

**Comment by shreyaansjain06** [comment]
yes that looks good

**Comment by vinzvinci** [comment]
You can use the magic words: **Lorem ipsum** :D

**Comment by shreyaansjain06** [comment]
I have put you can check previous figma link i share

**Comment by vinzvinci** [comment]
Yes, u can use that logo @shreyaansjain06.

U can make 4 cards for now.

1 for Dev Protocol
3 for random

**Comment by shreyaansjain06** [comment]
https://www.figma.com/file/P8ytuqbDBEgsSEJILvsgvV/Untitled?node-id=2%3A77
can anyone check and give their feedback

**Comment by vinzvinci** [comment]
Hi, I'm asking edit request, pls accept

**Comment by shreyaansjain06** [comment]
ok

**Comment by krishguptadev** [comment]
> Should i make community partners page using jekyll or i can makr with html css

We already using 11ty, You can read the docs of 11ty here: https://www.11ty.dev/docs/

**Comment by krishguptadev** [comment]
Why is their a horizontal scrollbar?
